CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## 2.29.1 / 2021-08-11
+
+* [BUGFIX] tsdb: align atomically accessed int64 to prevent panic in 32-bit
```

Comment:
We should specify that this also affects ARM and 386.

On another note, I don't think it was 32-bit that caused the issue on the Raspberry Pi. Models 3 and 4 are 64-bit (although Raspbian is still by default 32-bit which might matter) and the bug notes specify 32-bit _MIPS_ as the issue, which I believe the Pi is not, given that it has an ARM processor with uses an ARM instruction set.

Reply:
On [ARM], [386], and [32-bit MIPS]

ARM is 32 bit. it is not the *actual* processor that counts, it is the GOARCH.